plugins/recaptcha/class.recaptcha.plugin.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+/**
+ * @copyright 2010-2016 Vanilla Forums Inc
+ * @license Proprietary
+ */
+
+$PluginInfo['recaptcha'] = array(
+    'Name' => 'Recaptcha Support',
```

Comment:
They call it `reCAPTCHA` so let's stick with that stupid capitalization in user-facing labels.